### PR TITLE
Dockerfile updated with all the requirements specified by the Red Hat Container Certification

### PR DESCRIPTION
**Pull request by manojjahgirdar**

### CODE REVIEW

File: Dockerfile copy [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ibmgaragecloud/gradle:jdk11 AS builder
```
Comment (csantanapr) [inline Dockerfile copy:1]: remove this file?

File: manoj [hunk 2]
```diff
@@ -0,0 +1 @@
+manoj
```
Comment (csantanapr) [inline manoj:1]: what is this file? remove?

File: user.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+manoj
```
Comment (csantanapr) [inline user.txt:1]: what is this file? remove?

**Comment by jjasghar** [comment]
My gut tells me this was an automated PR. It should only be one commit, and like 2 files? (why is the license in here too?)

File: Dockerfile copy (hunk 1 above)
Comment (manojjahgirdar) [reply Dockerfile copy:1]: yes I noticed this is no longer used, I'll delete it, Thanks.

File: manoj (hunk 2 above)
Comment (manojjahgirdar) [reply manoj:1]: oops, I'll delete this as well

File: user.txt (hunk 3 above)
Comment (manojjahgirdar) [reply user.txt:1]: oops, I'll delete this as well

**Comment by manojjahgirdar** [comment]
I'll take down this PR as I need to work on the dockerfile a bit, also I have made a note of the review comments will fix them as well. Thanks.

**Pull request closed without merging**